src/models/node.js

```diff
@@ -1540,94 +1534,42 @@ const Node = {
   },
 
   /**
-   * Split a node by `path` at `offset`.
+   * Split a child node by `index` at `position`.
    *
-   * @param {Array} path
-   * @param {Number} offset
+   * @param {Number} index
+   * @param {Number} position
    * @return {Node}
    */
 
-  splitNode(path, offset) {
-    let base = this
-    const node = base.assertPath(path)
-    let parent = base.getParent(node.key)
-    const isParent = base == parent
-    const index = parent.nodes.indexOf(node)
-
-    let child = node
+  splitNode(index, position) {
```

Comment:
A few places still call `splitNode` with a path (array) instead of an index. This throws an exception when you tried to do `Cmd-C` (Copy command) on the editor (observable in the rich text example on this branch).

Like here:
https://github.com/ianstormtaylor/slate/pull/836/files#diff-82007cf8e57fb38e037a7553af96e41fR1533